tests/integration-tests/tests/budgets/test_budgets/test_update_budget_properties/budget_with_fleet_stop.yaml

```diff
@@ -0,0 +1,44 @@
+Image:
```

Comment:
Only the test config is changed, is there any change in the integration test code?

Reply:
Updated! Integration tests code included